modules/mono/glue/GodotSharp/GodotSharp/Core/Transform2D.cs

```diff
@@ -355,11 +355,45 @@ public Transform2D Translated(Vector2 offset)
         /// </summary>
         /// <param name="v">A vector to transform.</param>
         /// <returns>The transformed vector.</returns>
+        [Obsolete("Xform is deprecated. Use the multiplied(Transform2D * Vector2) instead.")]
         public Vector2 Xform(Vector2 v)
         {
             return new Vector2(Tdotx(v), Tdoty(v)) + origin;
         }
 
+        /// <summary>
+        /// Returns a Rect2 transformed (multiplied) by this transformation matrix.
+        /// </summary>
+        /// <param name="r">A Rect2 to transform.</param>
+        /// <returns>The transformed Rect2.</returns>
+        [Obsolete("Xform is deprecated. Use the multiplied(Transform2D * Rect2) instead.")]
+        public Rect2 Xform(Rect2 r)
```

Comment:
We don't need to add a method that is immediately obsolete. We only need the operators.

Reply:
Yes, if this is for master (4.0-dev) then just remove the method. For 3.x the obsolete attribute is good.